File: Web/RMA-Web-Shell/projects/admin/src/app/campaign-manager/shared/entities/target-audience.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class TargetAudience extends BaseClass {
    campaignId: number;
    policyId: number;
    policyNo: string;
    clientTypeId: number;
    itemType: string;
    itemId: number;
    name: string;
    email: string;
    memberNumber: string;
    idNumber: string;
    mobileNumber: string;
    postalAddress: string;
    unsubscribed: boolean;
    unsubscribedAll: boolean;
}

File: Service/Admin/RMA.Service.Admin.CampaignManager.Contracts/Entities/TargetAudience.cs
using RMA.Common.Entities;

namespace RMA.Service.Admin.CampaignManager.Contracts.Entities
{
    public class TargetAudience : AuditDetails
    {
        public int CampaignId { get; set; }
        public string ItemType { get; set; }
        public int ItemId { get; set; }
        public string Name { get; set; }
        public int ClientTypeId { get; set; }
        public string IdNumber { get; set; }
        public string MemberNumber { get; set; }
        public string RegistrationNumber { get; set; }
        public string Email { get; set; }
        public string MobileNumber { get; set; }
        public bool Unsubscribed { get; set; }
        public bool UnsubscribedAll { get; set; }

        public override string ToString()
        {
            return $"TargetAudience: {CampaignId} {ItemType} {ItemId}";
        }
    }
}